tests/arch/arm/arm_tz_wrap_func/src/README.txt

```diff
@@ -0,0 +1,44 @@
+Title: Test to verify the __TZ_WRAP_FUNC() macro.
+
+Description:
+The test verifies that:
+ - The wrapper functions are correctly called.
+ - The arguments are passed to the wrapped function.
+ - The return value from the wrapped function is correctly returned from the
+   wrapper function.
+
```

Comment:
Can you make a note that although the wrap framework is added for TrustZone purpose in Non-Secure builds, it is not restricted under #ifdef CONFIG_TRUSTED_EXECUTION_NONSECURE=y and that's why it is tested outside this restriction.
:)

Reply:
Added some text, see if it needs more.

Reply:
No, it looks sufficient